saiph/projection.py

```diff
@@ -31,9 +32,6 @@ def fit(
     Returns:
         model: The model for transforming new data.
     """
-    if isinstance(nf, str):
-        raise ValueError("nf=all is deprecated. Use None instead.")
-
```

Comment:
I am going to release a new minor version after this, as modifying seeding behavior is a breaking change IMO, so I considered this check to be deprecated.

Reply:
I guess you've checked if there are any other breaking changes to include.

Reply:
> I guess you've checked if there are any other breaking changes to include.

No